testing/src/main/java/io/stargate/it/http/docsapi/BaseDocumentApiV2Test.java

```diff
@@ -535,6 +535,15 @@ public void testPutNullsAndEmpties() throws IOException {
         .isEqualTo(wrapResponse(OBJECT_MAPPER.createArrayNode(), "4", null));
   }
 
+  @Test
+  public void testPutSimpleArray() throws IOException {
```

Comment:
should we consider adding other edge cases here? 

* I guess normal string is a valid JSON, should we test with string, blank string empty string?
* same with numbers, booleans
* empty documents (I guess this is not allowed at the moment)

Reply:
These are all disallowed, and tested by [testInvalidPuts](https://github.com/stargate/stargate/blob/bc9b7f509fb498044353e5d6ff63e39e2d720936/testing/src/main/java/io/stargate/it/http/docsapi/BaseDocumentApiV2Test.java#L665)